core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2207,14 +2207,27 @@ private[spark] object Utils extends Logging {
   }
 
   /**
+   * Return whether it is local mode
+   */
+  def isLocal(conf: SparkConf): Boolean = {
```

Comment:
It is just called "isLocal" and is a function of `SparkConf`. _What_ is local? why be opaque -- "isLocalMaster"? hm, does it even belong as a helper in `SparkConf`? (I don't feel strongly about that.)

You're already writing "Returns .."; why not just use the actual scaladoc tag?